plugins/zoom/icon_zoom/util/event.py

```diff
@@ -0,0 +1,30 @@
+from hashlib import sha1
+
+
+class Event(object):
+
+    # pylint: disable=redefined-builtin
+    def __init__(self, client_type: str, email: str, ip_address: str, time: str, type: str, version: str):
+        self.client_type = client_type
+        self.email = email
+        self.ip_address = ip_address
+        self.time = time
+        self.type = type
+        self.version = version
+
+    def __eq__(self, other):
+        return (
+            self.client_type == other.client_type
+            and self.email == other.email
+            and self.ip_address == other.ip_address
+            and self.time == other.time
+            and self.type == other.type
+            and self.version == other.version
+        )
+
+    def sha1(self):
+        m = sha1()  # nosec B303
+        for k, v in self.__dict__.items():
+            m.update(f"{k}{v}".encode("utf-8"))
+
+        return m.hexdigest()
```

Comment:
I don't like single letter variables at all. But it's up to you 😄 
```suggestion
        hash_ = sha1()  # nosec B303
        for key, value in self.__dict__.items():
            hash_.update(f"{key}{value}".encode("utf-8"))
        return hash_.hexdigest()
```
or something like that